src/pages/resources/index.mdx

```diff
@@ -254,17 +254,6 @@ Carbon for IBM.com delivers the [IBM Design Language](https://www.ibm.com/design
 
 ![Sketch icon](../../images/bee_icon.svg)
       
-</ResourceCard>
-  </Column>
-  
-  <Column colMd={4} colLg={4} noGutterSm>
-<ResourceCard
-  subTitle="Review design on Wake"
-  href="https://ibm.wake.com/space/37"
->
-
-![Sketch icon](../../images/wake_logo.svg)
-      
 </ResourceCard>
```

Comment:
Hey Jenny, you gotta remove this line too.

Reply:
@shixiedesign It's fine, you can see above in on line 257 and 258 the
```
</ResourceCard>	
  </Column>
   ...
```
lines were removed, so we need to keep this `</ResourceCard></Column>`

It's the same if we didn't remove the first two and removed it on lines 268 & 270

Reply:
Thanks!